pkgs/development/interpreters/python/hooks/default.nix

```
@@ -230,6 +230,29 @@ in
     } ./python-recompile-bytecode-hook.sh
   ) { };
 
+  pythonRelaxBuildDepsHook = callPackage (
+    {
+      makePythonHook,
```

```suggestion
      makePythonHook,
      writeTextFile,
```

BTW, hope that the use of `packaging` and `tomlkit` won't cause circular dependency in the future.